styles/styles.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  margin: 0 auto;
+  max-width: 800px;
+  padding: 20px;
+  background: linear-gradient(to right, #e66465, #9198e5);
```

Comment:
you are giving every element a background gradient by hand - you could just add it to your body once in the globalStyles